app/grandchallenge/products/utils.py

```diff
@@ -78,10 +80,9 @@ def _create_company(self, row):
         c.email = row["Email address (public)"]
         c.description = row["Company description"]
         c.description_short = self._split(row["Company description"], 200)
-        image_name = row["Company name"]
-        for ch in [" ", ".", "-", ","]:
-            image_name = image_name.replace(ch, "")
-        img_file = self.images_path.glob(f"**/logo/{image_name.lower()}.*")
+        slug = slugify(row["Company name"])
+        c.slug = slug
+        img_file = self.images_path.glob(f"**/logo/{slug}.*")
```

Comment:
You might have to change the naming of your images now, as slugify replaces spaces with dashes, whereas you removed them before.

Reply:
How do i know when this code has been deployed? Cause then I need to quickly change the database. If I do it now already it won't work for the mean time..

Reply:
You can check the status on grafana, ask @pkcakeout for an account.

Reply:
Notice the images don't even fail right away (thought they would). So not that much of a hurry after all. Made sure i changed the logo-names to slug for the next database upload :ok_hand: